src/@types/token.d.ts

```diff
@@ -0,0 +1,3 @@
+export type Token = {
```

Comment:
StyleDictionary exports some type definitions, could we pull from there instead? Also, this folder is called `@types`. Is there a relation to the `@types` package? I'd recommend we move this into `nameTransformer.ts` for now and let `tsc` create the `d.ts` during compile time.

Reply:
Good one, I'll change it, it is `TransformedToken`